packages/ui-library/babel.config.js

```diff
@@ -1,3 +1,20 @@
-module.exports = {
-	presets: [ "@yoast/babel-preset" ],
+module.exports = ( api ) => {
+	api.cache.using( () => process.env.NODE_ENV === "test" );
+
+	return {
+		presets: [
+			[
+				"@babel/preset-env",
+				{
+					modules: api.env( "test" ) ? "auto" : false,
```

Comment:
What's the difference between `process.env.NODE_ENV === "test"` and `api.env( "test" )`?

Reply:
So should we then use `api.env('test')` for `api.cache.using` as well?

Reply:
Actually, reading https://babeljs.io/docs/config-files#apienv
> This function internally makes use of [api.cache](https://babeljs.io/docs/config-files#apicache) mentioned above to ensure that Babel is aware that this build depends on a specific envName. You should not use it alongside with api.cache.forever() or api.cache.never().

Replacing the need to specify `api.cache` in the first place 😄